guides/doc-Upgrading_and_Updating/topics/preparing_to_migrate_pulp_content.adoc

```diff
@@ -52,14 +52,16 @@ double in size before starting the migration process.
 Check the size of /var/lib/pulp/published with 'du -sh /var/lib/pulp/published/'
 [OK]
 ----
-ifdef::satellite[]
+
 . Update the file permissions before upgrading {ProjectServer} using the following command:
 +
 [options="nowrap", subs="verbatim,quotes,attributes"]
 ----
 # {foreman-maintain} content prepare
 ----
-endif::[]
++
+. As part of the final step of `{foreman-maintain} content prepare`, {Project} checks whether any content units are unmigrated.
+If `{foreman-maintain} content migration-stats` identifies corrupted or missing content, you might see something similar to the following:
```

Comment:
In that case, it's not a separate step, but only an explanation for the preceding one. I would suggest something like:

```
By the end of its run, the command checks whether any content units are unmigrated.
If the command identifies corrupted or missing content, you might see something similar to the following in the output:
```

@jlsherrill How does that sound?

Reply:
that sounds great @Lennonka !!

Reply:
@ianf77 Can you please use this and fine tune it language-wise? :)